### PR TITLE
Dumpable component to use a doafter to empty storage into a disposal unit, placeable surface, or the ground

### PR DESCRIPTION
**Pull request by Elijahrane**
## About the PR 
Janitors and botanists rejoice.

**Changelog**

:cl: Rane
- add: The trash bag and plant bag can now be directly emptied into disposal units, onto tables, or on the ground with an alternative verb.

### CODE REVIEW

**Comment by mirrorcult** [comment]
the 'into a disposal unit/surface' part can probably use a utility verb. in fact i thought a utility verb already existed for this

**Comment by Elijahrane** [comment]
Added utility verbs

File: Content.Server/Storage/EntitySystems/DumpableSystem.cs [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+            {
+                Dumpable = dumpable;
+            }
+        }
+
+        private sealed class DumpCompletedEvent : EntityEventArgs
+        {
+            public EntityUid User { get; }
+            public EntityUid? Target { get; }
+            public DumpableComponent Dumpable { get; }
```
Comment (mirrorcult) [inline Content.Server/Storage/EntitySystems/DumpableSystem.cs:182]: dont store components on these events, idk if its even used

**Review by mirrorcult — CHANGES_REQUESTED** [review]

**Comment by Elijahrane** [comment]
Fixed conflicts and addressed review

**Comment by mirrorcult** [comment]
you have added a 30 thousand line [mono_crash.2536d94da2.0.json](https://github.com/space-wizards/space-station-14/pull/7792/commits/94b8cac12012e1d27b175aecff1c41300f9262b5#diff-a9182f7b0d3ddfc558e81cc8d20fe6b6599235c68d9797940a27ceed5639866a)

**Comment by Elijahrane** [comment]
gotta get my loc metrics up

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**